lib/read-wasm.js

```diff
@@ -0,0 +1,40 @@
+if (typeof fetch === "function") {
+  // Web version of reading a wasm file into an array buffer.
+
+  let mappingsWasmUrl = null;
+
+  module.exports = function readWasm() {
+    if (typeof mappingsWasmUrl !== "string") {
+      throw new Error("You must provide the URL of lib/mappings.wasm by calling " +
+                      "SourceMapConsumer.initialize({ 'lib/mappings.wasm': ... }) " +
```

Comment:
Would it be crazy to just have the wasm be a giant string constant in some `.js` file?

Reply:
It would bloat the code size, which I took quite a bit of effort in keeping small.